src/logic/application-code/python/application-code.test.ts

```diff
@@ -35,8 +36,8 @@ from one_sdk import OneClient, PerformError, UnexpectedError
 load_dotenv()
 
 client = OneClient(
-  # Optionally you can use your OneSDK token to monitor your usage. Get one at https://superface.ai/app
-  # token =
+  # ${ONESDK_TOKEN_COMMENT}
```

Comment:
issue: there should be the actual comment and not env var as it is in case of js code. Just do not know how why is this passing the test.

Reply:
You're right, fixing it